annotation.go

```diff
@@ -99,6 +112,148 @@ func (a *SchemaAnnotation) Decode(o interface{}) error {
 	return json.Unmarshal(buf, a)
 }
 
+// CreateGroups returns a creation groups schema-annotation.
+func CreateGroups(gs ...string) SchemaAnnotation {
+	return SchemaAnnotation{CreateGroups: gs}
+}
+
+// ReadGroups returns a read groups schema-annotation.
+func ReadGroups(gs ...string) SchemaAnnotation {
+	return SchemaAnnotation{ReadGroups: gs}
+}
+
+// UpdateGroups returns an update groups schema-annotation.
+func UpdateGroups(gs ...string) SchemaAnnotation {
+	return SchemaAnnotation{UpdateGroups: gs}
+}
+
+// ListGroups returns a list groups schema-annotation.
+func ListGroups(gs ...string) SchemaAnnotation {
+	return SchemaAnnotation{ListGroups: gs}
+}
+
+type PolicyConfig uint
+
+const (
+	Create PolicyConfig = iota
+	Read
+	Update
+	Delete
+	List
+)
```

Comment:
It feels to me that this type should be named something like `Op`, `Operation` or `HandlerType`, and the `Expose`/`Exclude` options should be named as `PolicyConfig`. Also, why do you need the `policy` package?

Reply:
I had some cyclic dependencies before refactoring and forget to dissolve this one. Thanks!